Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -174,6 +162,22 @@ private void HeaderClicked(ObjectsTabHeader.Header header)
         RefreshObjectList();
     }
 
+    private void RefreshButtonPressed(BaseButton.ButtonEventArgs args)
+    {
+        RefreshObjectList();
+    }
```

Comment:
This one can probably be an anonymous function. Doesn't matter that much though.